tools/CopyrightDateTool.scala

```diff
@@ -154,18 +152,16 @@ object CopyrightDateTool {
       println("Oops...We have some error dates")
     }
     else println ("Everything is ok")
-    check
+    error_files.nonEmpty
   }
 
   def main(args: Array[String]) {
     if (args.length != 1) println(usage)
```

Comment:
If line 159 is left in then the usage details will be printed twice when the number of args is <> 1